manim/mobject/text/text_mobject.py

```diff
@@ -166,9 +166,12 @@ def __init__(
         lines_str_list = lines_str.split("\n")
         self.chars = self._gen_chars(lines_str_list)
 
-        self.lines = [list(self.chars), [self.alignment] * len(self.chars)]
-        self.lines_initial_positions = [line.get_center() for line in self.lines[0]]
-        self.add(*self.lines[0])
+        # self.lines[0] -> self.lines_chars
+        # self.lines[1] -> self.lines_alignment
+        self.lines_chars = list(self.chars)
+        self.lines_alignment = [self.alignment] * len(self.chars)
```

Comment:
Minor nitpick, but I believe that a name like `self.lines_alignments` or `self.line_alignments` with a plural on "alignments" reflects better the fact that it is a list of alignments.